samcli/lib/telemetry/event.py

```diff
@@ -145,6 +154,27 @@ def clear_trackers():
         with EventTracker._event_lock:
             EventTracker._events = []
 
+    @staticmethod
+    def send_events():
+        """Sends the current list of events via Telemetry."""
+        from samcli.lib.telemetry.metric import Metric  # pylint: disable=cyclic-import
+
+        with EventTracker._event_lock:
+            if not EventTracker._events:  # Don't do anything if there are no events to send
+                return
+
+            telemetry = Telemetry()
+
+            try:
+                metric = Metric("events")
+                msa = {}
+                msa["events"] = [e.to_json() for e in EventTracker._events]
+                metric.add_data("metricSpecificAttributes", msa)
+                telemetry.emit(metric)
+                EventTracker._events = []  # Manual clear_trackers() since we're within the lock
+            except RuntimeError:
+                LOG.debug("Unable to find Click Context for getting session_id.")
```

Comment:
It seems we copy pasted this from some metric methods that use click context. Since we're not using it here, I believe we actually don't need the try and catch

Reply:
Right, just double checked and none of the methods in the try-catch throw anything, so it should be safe to remove!